assets/js/components/board.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-plusplus */
+const board = (() => {
+  let grid;
+  const returnGrid = () => grid;
+  const setCell = (index, symbol) => {
+    grid[index] = symbol;
+  };
+  const boardIsFull = () => {
+    for (let i = 0; i < grid.length; i++) {
+      if (grid[i] === null) {
+        console.log('i am here');
```

Comment:
Remove `console.log` it affects the performance of the code.

Reply:
Delated console log from all files.